config/configuration.toml.in

```diff
@@ -1,6 +1,6 @@
 # XXX: Warning: this file is auto-generated from file "@CONFIG_IN@".
 
-[hypervisor.qemu-lite]
+[hypervisor.q35]
```

Comment:
In that case, looking at what we do in virtcontainers, I think `qemu-system-x86_64` should end up being the `q35` version and `qemu-lite-system-x86_64` will obviously continue to be the `qemu-lite` version.